cmd/quadlet/main.go

```diff
@@ -160,7 +160,7 @@ func appendSubPaths(dirs []string, path string, isUserFlag bool, filterPtr func(
 }
 
 func nonNumericFilter(_path string, isUserFlag bool) bool {
-	// when running in rootless, only recrusive walk directories that are non numeric
+	// when running in rootless, only recursive walk directories that are non numeric
```

Comment:
This entire comment is hard to parse. Maybe "...only recurse into directories..."? Or "...only walk directories...", with the assumption that `walk` implies recursion?

Also, `non-numeric` is a compound adjective and should be hyphenated. Or written as "NOT numeric."

Also, punctuation. If I'm reading the comment correctly, maybe it should read "...are non-numeric SO AS TO ignore ..."?